Python/Watermark_Images/watermark_logo.py

```diff
@@ -0,0 +1,39 @@
+import cv2
+import numpy as np
+
+#importing the main image
+pImg = '/content/food.png' #path to main image

```

Comment:
Is it possible to pass the image file path dynamically?

Reply:
Done